third_party/tutorial.proto

```diff
@@ -88,17 +89,21 @@ message InfoBox {
 };
 
 message CodeBlock {
-  repeated InlineContent inline_contents = 1;
-  string code_location = 2;
+  string text = 4;
   string language_hint = 3;
+
+  reserved 1, 2;
+  reserved "inline_contents", "code_location";
 };
 
 message ListStep {
-  repeated Content contents = 1;
+  repeated InlineContent contents = 2;
+
+  reserved 1;
```

Comment:
Add reserved name

Reply:
also, this seemed to be from the closed pr... forgot to clean the inherited code.